images/router/egress/README.md

```diff
@@ -32,6 +32,9 @@ spec:
       value: 192.168.12.1
     - name: EGRESS_DESTINATION
       value: 203.0.113.25
+    # Set mtu size, optional, 1450 by default.
```

Comment:
Because as an Openshift user, you may want to change the MTU size or not, just to highlight that the MTU_SIZE is and optional env variable, unlike other env variables that are mandatory. 
But If you think Is better to get ride of the comments, Its ok.